### ISSUE TITLE
1010d/10-7959c Merge Staging Review finding: Link with incorrect accessible name

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#118209]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-10D CHAMPVA Benefits and 10-7959c OHI
**Feature:** Merged form solution

## Findings details
**VA.gov Experience Standard - issue:** [User can't complete a necessary flow or hits a dead end.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Introduction > "Start your form without signing in" has `aria-label="test-aria-label"` overriding the link's text value for screen readers. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="950" height="643" alt="link accessible name" src="https://github.com/user-attachments/assets/cbaf1002-40dd-4a9e-836f-345c84be3a6a" />

</details>

## Recommended action

Remove the `aria-label`

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG22/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by andrea-merrill** [comment]
this has been resolved in this ticket [[FE] 10-10d/10-7959c - Form merge: Swap out login alert on Introduction page](https://github.com/department-of-veterans-affairs/va.gov-team/issues/117987)

fysa: @ManOnTheMooney
